### PR TITLE
Fixes to known issues with the 2.2.6 docs. Does not fix gsets menu issue

### PR DESCRIPTION
**Pull request by Bob-The-Marauder**
Extracted these issues from within #riak-docs in Slack as we were originally noting them in there until gsets hit.

### CODE REVIEW

**Review by daniel020403 — APPROVED** [review]
LGTM!

**Review by pjaclark — CHANGES_REQUESTED** [review]
File paths are wrong. https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/bionic64/ not https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/bionic_64/

Similar for other paths.

File: content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md [hunk 1]
```diff
@@ -105,28 +105,28 @@ for the target platform:
 #### Ubuntu Bionic Beaver (18.04)
 
 ```bash
-wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/bionic/riak_2.2.6-1_amd64.deb
+wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/bionic_64/riak_2.2.6-1_amd64.deb
```
Comment (pjaclark) [inline content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md:108]: bionic64

File: content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md [hunk 2]
```diff
@@ -105,28 +105,28 @@ for the target platform:
...
 sudo dpkg -i riak_2.2.6-1_amd64.deb
 ```
 
 #### Ubuntu Xenial Xerus (16.04)
 
 ```bash
-wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/xenial/riak_2.2.6-1_amd64.deb
+wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/xenial_64/riak_2.2.6-1_amd64.deb
```
Comment (pjaclark) [inline content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md:115]: xenial64

File: content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md [hunk 3]
```diff
@@ -105,28 +105,28 @@ for the target platform:
...
 sudo dpkg -i riak_2.2.6-1_amd64.deb
 ```
 
 #### Ubuntu Trusty Tahr (14.04)
 
 ```bash
-wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/trusty/riak_2.2.6-1_amd64.deb
+wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/trusty_64/riak_2.2.6-1_amd64.deb
```
Comment (pjaclark) [inline content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md:122]: trusty64

File: content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md [hunk 4]
```diff
@@ -105,28 +105,28 @@ for the target platform:
...
 sudo dpkg -i riak_2.2.6-1_amd64.deb
 ```
 
 #### Ubuntu Precise Pangolin (12.04)
 
 ```bash
-wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/precise/riak_2.2.6-1_amd64.deb
+wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/precise_64/riak_2.2.6-1_amd64.deb
```
Comment (pjaclark) [inline content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md:129]: precise64

**Review by pjaclark — CHANGES_REQUESTED** [review]
Path changes.

File: content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md (hunk 1 above)
Comment (Bob-The-Marauder) [reply content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md:108]: that's weird - I went through and corrected tham all. Nevertheless, will accept your identical changes

Comment (Bob-The-Marauder) [reply content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md]: Oh, just realised you didn't make changes. I have since re-edited the file

**Review by pjaclark — APPROVED** [review]

**Comment by pjaclark** [comment]
Changes look good to me.

**Comment by pjaclark** [comment]
Merged. @Bob-The-Marauder : ok to delete branch?

**Pull request merged**